testing/regress/ecl/mysqlembed.ecl

```diff
@@ -80,6 +80,10 @@ dataset(childrec) testMySQLDS2() := EMBED(mysql : server(myServer),user(myUser),
   SELECT [] from tbl1 where u1='Straße';
 ENDEMBED;
 
+streamed dataset(childrec) testMySQLDS2a() := EMBED(mysql : server(myServer),user(myUser),database(myDB),PROJECTED('[]'), activity, local(true))
+  SELECT [] from tbl1 where (NOT __LOCAL__) OR (__NUMSLAVES__<2) OR (value = __SLAVE__+1);
```

Comment:
Will the last element  work on a 4 way thor?

Reply:
I think so, as there are only 2 items in the dataset.